lib/tasks/BackbeatTask.js

```diff
@@ -31,11 +31,33 @@ class BackbeatTask {
         const startTime = Date.now();
         const self = this;
 
+        // FIXME workaround for S3C-4457:
+        //
+        // It seems the S3 client may call its callback multiple times
+        // in an unknown corner case (the callback passed to the
+        // send() function, like in ReplicateObject._setupRolesOnce()).
+        //
+        // Until we find the root cause, we catch duplicate calls and
+        // log them instead of crashing the process with an exception
+        // raised from the async module.
+        //
+        let cbCalled = false;
```

Comment:
Should we use `jsutil.once`?

https://github.com/scality/Arsenal/blob/a1c0dd24723be7f63a91056df98ccd8a256e4ef3/lib/jsutil.js#L20

Reply:
Not a good fit here, we want to log a warning when the duplicate happens, which jsutil doesn't do (it considers it a "normal" case).